roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -63,7 +63,7 @@
     # source ~/overcloudrc;
     {{ openstack_cmd }} metric list
   register: result
-  failed_when: result.rc >= 1
+  failed_when: '"ceilometer_cpu" not in result.stdout'
```

Comment:
The original didn't do much checking. I think this condition is better as it does what's written in the task name.